flask_jwt_extended/view_decorators.py

```diff
@@ -196,12 +196,17 @@ def _decode_jwt_from_json(token_type):
     return encoded_token, None
 
 
-def _decode_jwt_from_request(token_type):
+def _decode_jwt_from_request(token_type, locations):
     # All the places we can get a JWT from in this request
     get_encoded_token_functions = []
```

Comment:
I decided to not add any validation of the `locations` argument as it would have to execute on every request. I think it is safe to assume the developer will use valid values as per coming documentation. Seemed different enough from having to protect against config file errors. Figured I would point this out so you can consider what really is right to do.

Reply:
I'm good with the approach you've taken here. If it ends up being confusing for users we can always add some validation back in later :+1: